src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -21,5 +21,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_STATUS = new Prefix("S/");
     public static final Prefix PREFIX_START = new Prefix("ST/");
     public static final Prefix PREFIX_END = new Prefix("EN/");
+    public static final Prefix PREFIX_LINK = new Prefix("L/");
```

Comment:
In line with our next changes to make prefixes case insensitive, we will be making all prefix arguments in this class small letters. Please change it to `l/` instead of `L/` currently.